models/conversation_message.py

```
@@ -0,0 +1,60 @@
+    """Represents a message (either from an LLM or a user) in a conversation.
+
+    Attributes:
+        role (str): Denotes the provenance of a message (e.g., "user", "system").
+        content (str): The content of a message (i.e., the actual prompt).
+    """
+
+    role: str
+    content: str
+
+    @abc.abstractmethod
+    def __init__(self):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)